### PR TITLE
Automatic client CDN dependency updates, maybe

### PR DESCRIPTION
**Pull request by bakert**

### CODE REVIEW

**Comment by bakert** [comment]
@silasary does what I'm doing here with the subprocess git commands look sane? It works on local but I really have no idea what it's going to do on prod ;)

File: maintenance/client_dependencies.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+PATH = 'shared_web/templates/jsdependencies.mustache'
+
+def run() -> None:
+    tags = [fetch_script_tag(library) + '\n' for library in get_dependencies()]
+    output = ''.join(tags)
+    write_dependencies(output)
+    subprocess.call(['git', 'add', PATH])
+    if subprocess.call(['git', 'commit', '-m', '"Update client dependencies."']) == 0:
+        subprocess.call(['git', 'push'])
+        subprocess.call(['hub', 'pull-request', '-b', 'master', '-m', 'Update client dependencies.' '-f'])
```
Comment (bakert) [inline maintenance/client_dependencies.py]: This is the part I'm really unsure of.

Comment (silasary) [reply maintenance/client_dependencies.py]: This should be an ad-hoc, and should never run on prod.
I can wrap it into the jenkins stuff

Comment (bakert) [reply maintenance/client_dependencies.py]: Ah, yeah, that makes so much more sense.

Comment (bakert) [reply maintenance/client_dependencies.py]: 0292c632 does this.

**Pull request merged**